statistics/statistics.go

```diff
@@ -103,16 +89,12 @@ func (p *Plugin) calculateStats(buff interface{}, startTime time.Time, stopTime
 		}
 	}
 
-	statList := [...]string{"Count", "Mean", "Median", "Standard Deviation", "Variance", "95%-ile", "99%-ile", "Minimum", "Maximum", "Range", "Mode", "Kurtosis", "Skewness", "Sum", "Trimean"}
-
+	statList := [...]string{"Count", "Mean", "Median", "Standard Deviation", "Variance", "95%-ile", "99%-ile", "2%-ile", "9%-ile", "25%-ile", "75%-ile", "91%-ile", "98%-ile", "Minimum", "Maximum", "Range", "Mode", "Kurtosis", "Skewness", "Sum", "Trimean", "Quartile_Range"}
```

Comment:
removing space between "standard deviation". Let's use all lower case. Be consistent between hyphen and underscore.